views/settings/page-sections/dashboard.php

```diff
@@ -24,75 +24,90 @@
 ?>
 <div id="<?php echo esc_attr( $data['id'] ); ?>" class="wpr-Page">
 	<div class="wpr-sectionHeader">
-		<h2 class="wpr-title1 wpr-icon-home"><?php echo $data['title']; ?></h2>
+		<h2 class="wpr-title1 wpr-icon-home"><?php echo esc_html( $data['title'] ); ?></h2>
 	</div>
 
 	<?php
-	$boxes = get_user_meta( $GLOBALS['current_user']->ID, 'rocket_boxes', true );
+		$rocket_boxes = get_user_meta( $GLOBALS['current_user']->ID, 'rocket_boxes', true );
```

Comment:
Could be nice to stop using this global, and use `get_current_user_id()` instead.